src/modules/fancyzones/lib/trace.cpp

```diff
@@ -77,7 +79,18 @@ void Trace::SettingsChanged(const Settings& settings) noexcept
         TraceLoggingBoolean(settings.zoneSetChange_flashZones, "FlashZonesOnZoneSetChange"),
         TraceLoggingBoolean(settings.zoneSetChange_moveWindows, "MoveWindowsOnZoneSetChange"),
         TraceLoggingBoolean(settings.overrideSnapHotkeys, "OverrideSnapHotKeys"),
-        TraceLoggingWideString(settings.zoneHightlightColor.c_str(), "ZoneHighlightColor"));
+        TraceLoggingBoolean(settings.appLastZone_moveWindows, "MoveWindowsToLastZoneOnAppOpening"),
+        TraceLoggingBoolean(settings.use_cursorpos_editor_startupscreen, "UseCursorPosOnEditorStartup"),
+        TraceLoggingWideString(settings.zoneHightlightColor.c_str(), "ZoneHighlightColor"),
+        TraceLoggingInt32(settings.zoneHighlightOpacity, "ZoneHighlightOpacity"),
+        TraceLoggingBoolean(editorHotkey.alt_pressed(), "HotkeyAltPressed"),
+        TraceLoggingBoolean(editorHotkey.ctrl_pressed(), "HotkeyCtrlPressed"),
+        TraceLoggingBoolean(editorHotkey.shift_pressed(), "HotkeyShiftPressed"),
+        TraceLoggingBoolean(editorHotkey.win_pressed(), "HotkeyWinPressed"),
+        TraceLoggingUInt32(editorHotkey.get_code(), "HotkeyCode"),
+        TraceLoggingWideString(editorHotkey.get_key().c_str(), "HotkeyKeyFromCode"),
```

Comment:
Let's concatenate the hotkeys entries into a single string, with basic formatting like:
```
alt:true,ctrl:false... etc.
```

Reply:
Sure, I've updated hotkey entry with string formatted like `alt:0, ctrl:0, shift:0, win:1, code:192 ...`